tests/testthat/public-api/xyz-r-and-rmd-dir/subdir/random-rmd-script-in-subdir.Rmd

```diff
@@ -0,0 +1,30 @@
+---
+title: "random-rmd-script"
+author: "Lorenz Walthert"
+date: "11/25/2017"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+1+1
```

Comment:
Do the tests actually test that styling is applied?

Reply:
I think we just check whether the file was named in the message about changed files. Should we test for the actual change?

Reply:
They don't check whether styling is applied, just which files are being named in the changed files message.  @lorenzwalthert suggested these tests so I kinda assumed we had our bases covered with actual styling tests elsewhere.